src/locales/cy/translation.json

```diff
@@ -178,13 +178,13 @@
       "paragraph1": "You need to prove your identity before you [doThing].",
       "paragraph2": "Your GOV.UK account will check your saved proof of identity to confirm it’s really you."
     },
-    "identityConfirmed": {
```

Comment:
was this linting?

Reply:
This is me being lazy and doing `cp src/locales/en/translation.json src/locales/cy/translation.json` after adding the new English content since the two files should be identical at the moment.